### PR TITLE
Move filesize to dev dependencies

### PR DESCRIPTION
**Pull request by realityking**
It's only used in the client

### CODE REVIEW

**Comment by TrySound** [comment]
Is client bundled before publishing?

File: package.json [hunk 1]
```diff
@@ -37,7 +37,6 @@
     "acorn-walk": "^8.0.0",
     "chalk": "^4.1.0",
```
Comment (alexander-akait) [inline package.json:38]: Can be replaced by `colorette` (light, fast and with simple API), already used in `webpack-cli@4`, so for developers which use `webpack-cli` and `webpack-bundle-analyzer` we will have less deps

**Comment by realityking** [comment]
@TrySound Yes: https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/master/package.json#L23

File: package.json (hunk 1 above)
Comment (realityking) [reply package.json:38]: Good point but something for a different PR :)

**Review by valscion — APPROVED** [review]
Cool :+1: Just checked that `mobx` is also only in devDependencies so client-only packages are OK to be specified only in devDependencies :+1:

**Pull request merged**